### ISSUE TITLE
Disable Video Plugins

### ISSUE THREAD

**Issue by BraunreutherA** [vacenz/last-draft#18]
Is it already possible to somehoe disable the video or / and image plugin? From reading the code, it seams they always get appended. 
Wouldn't you think it's better to give the user the freedom to decide if they are needed?

**Comment by StevenIseki** [comment]
Okay, that makes sense, I never thought that video and image would be optional but now you mention it, I can see why. 

What I might do is either move them out as `ld-video` and `ld-image` plugins, or I could allow users to import the `image` and `video` from last draft.

```js
import {image, video} from 'last-draft'
import todo from 'ld-todo'
import emoji from 'ld-emoji'
let plugins = [image, video, emoji, todo]
```

If anyone has a preference let me know, I will get this set up in a few days.

**Comment by BraunreutherA** [comment]
Hi I think importing them from last draft would be the easiest way. Making an own plugin-in just would reduce the file size. I would place them in last-draft as probably most people will use them.

**Comment by StevenIseki** [comment]
Actually I am just going to move video out to ld-video, so the image plugin will be the only default one, as it is a core plugin. This way its more consistent, and users won't have to worry about importing then adding those 2 differently, they just have to remember image plugin is built in, and all others are custom plugins. I cleaned up the importing ready to support this and added ld-video, so I will have it all ready and published by tomorrow in the next release.

**Comment by StevenIseki** [comment]
All fixed now in `v1.3.0` I moved video out to [ld-video](https://github.com/vacenz/ld-video) so it is now optional.

**Comment by StevenIseki** [comment]
Reopening this for a little while, so people can quickly see it and identify the issue if they used a previous version.
